Source/WebKit/Shared/Extensions/WebExtensionConstants.h

```diff
@@ -35,7 +35,7 @@ static constexpr size_t webExtensionDeclarativeNetRequestMaximumNumberOfEnabledR
 static constexpr size_t webExtensionDeclarativeNetRequestMaximumNumberOfDynamicAndSessionRules = 5000;
 
 // MARK: Storage constants.
-static constexpr double webExtensionUnlimitedStorageQuotaBytes = DBL_MAX;
+static constexpr double webExtensionUnlimitedStorageQuotaBytes = std::numeric_limits<double>::max();
```

Comment:
This seems unrelated to this patch, probably at least warrants a mention in the commit message.

Reply:
Will mention. Was just a drive-by.